packages/headless/src/app/search-engine/search-engine.ssr.ts

```diff
@@ -37,6 +41,12 @@ export type SearchEngineDefinitionOptions<
   TControllers extends ControllerDefinitionsMap<SearchEngine, Controller>,
 > = EngineDefinitionOptions<SearchEngineOptions, TControllers>;
 
+function isSearchCompletedAction(
```

Comment:
isSearchCompletedAction -> isSearchActionCompleted ?

Reply:
oh this is meant as a typeguard, checking whether the parameter is of type "SearchCompletedAction" essentially.

To make it more readable, I'll add a type alias called `SearchCompletedAction` which is a union of the fulfilled and rejected actions.